module-1/README.md

```diff
@@ -63,55 +117,255 @@ export const environment = {
+This command will initialise a new CDK application.
 
-Follow these minimum steps for working with the AWS Amplify CLI, or view the full [Getting Started guide](https://aws-amplify.github.io/amplify-js/media/quick_start?platform=purejs).
+### Implementation
+
+Now, let's implement the code to host our web application.  Open the `CDK` folder in VS Code.
+
+The standard structure of an AWS CDK application consists of a `bin` folder and a `lib` folder.
+
+* The `bin` folder is where we will define the entry point for the CDK application.
+* The `lib` folder is where we will define all our workshop infrastructure components.
+
+before we start implementing our code, let's get typescript to watch for file changes and compile the javascript whenever a change is detected by running the following command from the terminal.
+
+```bash
+npm run watch
+```
 
-* Install the AWS Amplify CLI:
-    * npm install -g @aws-amplify/cli
-    * amplify configure
-* Navigate to the `frontend` directory.
-* Run `amplify init` in the `frontend` directory and select the following options:
-    * Choose your default editor: `Visual Studio Code`
-    * Choose the type of app that you're building: `javascript`
-    * What javascript framework are you using: `angular`
-    * Source Directory Path: `src`
-    * Distribution Directory Path: `dist`
-    * Build Command: `npm run build -- --prod`
-    * Start Command: `npm start`
-* Run `amplify add hosting`
-    * Select `PROD (S3 with CloudFront using HTTPS)`
-    * Provide a [globally unique name](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html) for your hosting bucket
-* Run `amplify publish`
+#### Module 1 Files
+
+Next, Within the lib folder you should find a file with the default name of `cdk-stack.ts`.  Rename this file to `Module1Stack.ts`.
+
+Open the file `Module1Stack.ts` in VS Code and ename the class to `Module1Stack`, as illustrated below.
+
+```typescript
+import cdk = require('@aws-cdk/cdk');
+
+export class Module1Stack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // The code that defines your stack goes here
+  }
+}
+```
+
+Following the change of filename and classname, you should now update the references in the `cdk.ts` file, as such.
+
+```typescript
+#!/usr/bin/env node
+import 'source-map-support/register';
+import cdk = require('@aws-cdk/cdk');
+import { Module1Stack } from '../lib/Module1Stack';
+
+const app = new cdk.App();
+new Module1Stack(app, 'Module1');
+```
+
+Now let's go through defining the S3 and CloudFront infrastructure.
+
+#### Define the S3 bucket
+
+First of all, we are going to define our S3 bucket and define the web index document as 'index.html'
+
+```typescript
+const projectName = "mythical-mysfits";
+const bucket = new s3.Bucket(this, "Bucket", {
+  bucketName: projectName + "-frontend-" + cdk.Aws.accountId,
+  websiteIndexDocument: "index.html"
+});
+```
+
```

Comment:
Potentially, maybe you could go through a "cdk synth" and even "cdk deploy" cycle right now as a learning opportunity. It will be easier to explain this workflow with a small simple stack like you have at this point, and then you could also show "cdk diff" at the next stage.